packages/server/README.md

```diff
@@ -34,6 +34,7 @@ For local development, you'll only need to create a copy of `.env.example` and n
 You can do this via your system's file explorer or via the following terminal command while in the repo:
 
 ```console
+cd packages/server
 cp .env.example .env
```

Comment:
So the `local:start` now creates one for us if one isn't created yet, ya? If that's the case, should we note that this isn't necessary if you've already run `local:start` or recommend using that instead?

Reply:
For sure - I'll add a note